addons/website/views/res_config_settings_views.xml

```diff
@@ -20,23 +20,33 @@
         <field name="arch" type="xml">
             <xpath expr="//form" position="inside">
                 <app data-string="Website" string="Website" name="website" groups="website.group_website_designer">
-                    <setting type="header" string="Settings of Website">
+                    <setting
+                        type="header"
+                        string="Settings of Website"
+                        documentation="/applications/websites/website/configuration/multi_website.html"
+                    >
```

Comment:
@rdeodoo the doc link doesn't show up, is it because this setting has a specific class? How should I fix it? (or maybe I shouldn't and just remove the doc link ^^' )

Reply:
I guess, this `type="header"` is a very specific thing, used only in website and POS and a third one I believe.
It's probably not correctly supporting this kind of doc (as other setting blocks not converted by the framework recent refactoring)